disk/test_disk.py

```diff
@@ -0,0 +1,263 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+from nose.plugins.attrib import attr
+import re
+
+# 3p
+import mock
+
+# project
+from shared.test.common import AgentCheckTest, Fixtures
```

Comment:
I added some stuff to the Fixtures class in the `dd-agent` repo so you can use them from the agent. I think either is fine, you can pick your poison and we can consolidate later?